libs/small-steps/src/Control/State/Transition/Extended.hs

```diff
@@ -51,6 +51,7 @@ module Control.State.Transition.Extended (
   labeledPredE,
   ifFailureFree,
   whenFailureFree,
+  whenFailureFreeDefault,
```

Comment:
This needs a chnagelog entry and a minor bump in version.
Also `cardano-ledger-conway` will need a lower bound update on `small-steps`, since it now requires this function